examples/Thermostat/slice/Reading.slice

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Represents a snapshot of the thermostat state at a point in time.
+struct Reading {
+    /// The point in time where this Reading was taken.
+    timeStamp: WellKnownTypes::TimeStamp
+
+    /// The current temperature (in °F).
+    temperature: float32
+
+    /// The target temperature (in °F).
+    setPoint: float32
+
+    /// The state of the AC unit.
+    cooling: Stage
+}
+
+/// Represents the possible states of the AC unit.
+enum Stage : uint8 { Off, Stage1, Stage2 }
```

Comment:
It might be worth writing short doc comments for these enumerators.
Most people who don't obsess over A/C systems like us will be unfamiliar with "stages".

Reply:
Added enumerator doc-comments.